topics/admin/tutorials/job-destinations/tutorial.md

```diff
@@ -729,7 +729,7 @@ Lastly, we need to write a rule in TPV that will read the value of the job resou
 >         max_mem: 8
 >         params:
 >    -      native_specification: --nodes=1 --ntasks=1 --cpus-per-task={cores}
->    +      native_specification: --nodes=1 --ntasks=1 --cpus-per-task={cores} --time={params['walltime']}:00:00
+>    +      native_specification: --nodes=1 --ntasks=1 --cpus-per-task={cores} --time={entity.params.get('walltime')}:00:00
```

Comment:
This seems odd, if you anticipate walltime isn't set you would need to set a default, otherwise indexing via `[]` is what I'd use ... there's an obvious error if the value isn't set.

... also if `{cores}` works, i would keep it consistent and use `{params['walltime']}` ?

Reply:
Good point, I was testing the training on one of the EU training vms and was getting an error saying params was undefined. I'll post the logs in a couple of hours.